circle.yml

```diff
@@ -538,69 +566,22 @@ jobs:
     <<: *defaults
     parallelism: 5
     steps:
-      - attach_workspace:
-          at: ~/
-      - run:
-          command: yarn lerna exec --scope @packages/driver "yarn start"
-          background: true
-      - run:
-          command: yarn wait-on http://localhost:3500
-      - run:
-          command: |
-            CYPRESS_KONFIG_ENV=production \
-            CYPRESS_RECORD_KEY=$PACKAGES_RECORD_KEY \
-            yarn lerna exec --scope @packages/driver "yarn cypress:run --record --parallel --group 5x-driver-chrome --browser chrome"
-      - store_test_results:
-          path: /tmp/cypress
-      - store_artifacts:
-          path: /tmp/artifacts
-      - store-npm-logs
+      - run-driver-integration-tests:
+          browser: chrome
 
   # "driver-integration-tests-electron":
   #   <<: *defaults
   #   parallelism: 5
   #   steps:
-  #     - attach_workspace:
```

Comment:
I like how you removed commented out code too :) and inserted commented out NEW code

Reply:
gotta keep those comments up to date!